src/voice_annotation_tool/opened_project_frame.py

```diff
@@ -271,3 +302,23 @@ def import_profile_pressed(self):
     def mark_unchanged_pressed(self):
         for annotation in self.get_selected_annotations():
             self.project.mark_unchanged(annotation)
+
+    @Slot()
+    def record_pressed(self):
+        if not self.project.audio_folder or not self.project.audio_folder.is_dir():
+            return
+        if self.recorder.recorderState() == QMediaRecorder.StoppedState:
+            name = "000" + "".join(random.choices("0123456789abcdef", k=125))
```

Comment:
random is difficult, as it might happen at random, that the same value is chosen, which might lead to bugs.